File: Java/Events/GetFullPerformanceAvailabilityResult.java
package LTD_API;

import java.util.List;

public class GetFullPerformanceAvailabilityResult
{
    private int performanceId;
    private int eventId;
    private List<FullTicketInfo> availableTickets;
	
    public int getPerformanceId()
    {
        return performanceId;
    }
    public void setPerformanceId(int performanceId)
    {
        this.performanceId = performanceId;
    }

    public int getEventId()
    {
        return eventId;
    }
    public void setEventId(int eventId)
    {
        this.eventId = eventId;
    }

    public List<FullTicketInfo> getAvailableTickets()
    {
        return availableTickets;
    }
    public void setAvailableTickets(List<FullTicketInfo> availableTickets)
    {
        this.availableTickets = availableTickets;
    }
}

File: PHP/Events/GetFullPerformanceAvailabilityResult.php
<?php
namespace LTD_API;

class GetFullPerformanceAvailabilityResult
{
    private $performanceId;
    private $eventId;
    //array of FullTicketInfo
    private $availableTickets;
	
    public function getPerformanceId()
    {
        return $this->performanceId;
    }
    public function setPerformanceId($performanceId)
    {
        $this->performanceId = $performanceId;
    }

    public function getEventId()
    {
        return $this->eventId;
    }
    public function setEventId($eventId)
    {
        $this->eventId = $eventId;
    }

    public function getAvailableTickets()
    {
        return $this->availableTickets;
    }
    public function setAvailableTickets($availableTickets)
    {
        $this->availableTickets = $availableTickets;
    }
}
